core/modules/replica/ConfigApp.cc

```diff
@@ -83,11 +369,14 @@ ConfigApp::ConfigApp(int argc, char* argv[])
             enableServiceProvider,
             injectXrootdOptions
         ),
-        _log(LOG_GET("lsst.qserv.replica.ConfigApp")) {
+        _log(LOG_GET("lsst.qserv.replica.ConfigApp")),
+        _configUrl("mysql://qsreplica@localhost:3306/qservReplica") {
 
     parser().commands(
         "command",
-        {"DUMP",
+        {"MYSQL_CREATE", "MYSQL_SCHEMA_UPGRADE", "MYSQL_SCHEMA_DUMP", "MYSQL_TEST",
+         "SCHEMA_VERSION",
+         "DUMP",
```

Comment:
I don't think there is a standard for that. I've seen both. In the language of the parser that I have these are the so-called "commands" determining a sub-tree. The capitalization is just a convention I'm using throughout all applications of the Replication system, not just here. The general syntax is:
```
qserv-replica-some-app <command> <mandatary-parameters> <optional-parameters> <options-with-values> <flags>
```
That makes:
```
qserv-replica-some-app DUMP ...
qserv-replica-some-app MAKE ...
etc.
```

Reply:
You are right that there is no standard, but the is a lot of pre-existing art. All command line tools tools that I know (git, svn, docker, many more) use lower case for sub-commands. It would be less surprising for our users if we stick with common conventions in our tools as well.